packages/gmap-vue/src/components/heatmap-layer.js

```diff
@@ -0,0 +1,27 @@
+import mapElementFactory from '../factories/map-element'
+
+const props = {
+    options: {
+        type: Object,
+        twoWay: false,
+        default: () => {},
+    },
+    data: {
+        type: Array,
+        twoWay: true        
+    },
+};
+
+const events = [];
+
+/**
+ * @class Heatmap Layer
+ *
+ * Heatmap Layer class
+ */
+export default mapElementFactory({
+    mappedProps: props,
+    events: events,
+    name: 'heatmapLayer',
+    ctr: () => google.maps.visualization.HeatmapLayer
+});
```

Comment:
Picky: Don't forget to add a new line at the end of every file that you add or edit 😉